build/debian/owasp-zap.desktop

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env xdg-open
```

Comment:
It seems to be related to the .desktop file being executable.

https://unix.stackexchange.com/questions/294952/should-usr-bin-env-xdg-open-be-included-in-desktop-files